cmake/version.h.in

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#define MANIFOLD_VERSION_MAJOR @MANIFOLD_VERSION_MAJOR@
+#define MANIFOLD_VERSION_MINOR @MANIFOLD_VERSION_MINOR@
+#define MANIFOLD_VERSION_PATH @MANIFOLD_VERSION_PATCH@
```

Comment:
Typo: PATH -> PATCH

Reply:
Ups, I take full responsibility for that typo...